lightning/src/blinded_path/message.rs

```diff
@@ -31,6 +31,8 @@ use crate::util::ser::{FixedLengthReader, LengthReadableArgs, Writeable, Writer}
 use core::mem;
 use core::ops::Deref;
 
+use super::utils::Padding;
```

Comment:
Let's move this into the other import above.

Reply:
Addressed in [pr3177.09](https://github.com/shaavan/rust-lightning/commits/pr3177.09)
Thanks for the pointer!